pkg/controller/operators/olm/config.go

```diff
@@ -37,6 +37,26 @@ type operatorConfig struct {
 	configClient                 configv1client.Interface
 }
 
+func (o *operatorConfig) OperatorClient() operatorclient.ClientInterface {
```

Comment:
These chances go in conjunction with the OperatorConfig client interface on the plug-in package. This way the plug-in can have access to some of the configuration information